### ISSUE TITLE
Can't run setup

### ISSUE THREAD

**Issue by Safety1st** [Orckestra/C1-CMS-Foundation#663]
I use original distributive of v6.6 and when accessing site I get:
```Server Error in '/' Application.
Configuration Error
Description: An error occurred during the processing of a configuration file required to service this request. Please review the specific error details below and modify your configuration file appropriately. 

Parser Error Message: The value of the property 'type' cannot be parsed. The error is: The type initializer for 'Composite.Core.Types.CodeGenerationManager' threw an exception.

Source Error: 


An application error occurred on the server. The current custom error settings for this application prevent the details of the application error from being viewed remotely (for security reasons). It could, however, be viewed by browsers running on the local server machine.

Source File: D:\home\goldsrv.ru\httpdocs\App_Data\Composite\Composite.config    Line: 420 

Version Information: Microsoft .NET Framework Version:4.0.30319; ASP.NET Version:4.7.3282.0```

**Comment by ckeller81** [comment]
Good morning :)

Please check if you have installed .NET Framework Runtime 4.7.1 / 4.7.2 successfuly (get it from here: https://dotnet.microsoft.com/download/dotnet-framework/net472)

Also check that your IIS have activated the WebSockets Feature (added via Server Management > Manage Features).
Please read along the system requirements here: https://docs.c1.orckestra.com/Getting-started/System-Requirements to get all the software requirements right.

Hope that helps.

Best regrads and have a nice day,
Chris

**Comment by Safety1st** [comment]
That requirements are met.

**Comment by JustAndrei** [comment]
I'm experiencing the same problem

**Comment by JustAndrei** [comment]
Solved: it was necessary to provide write access to the app pool user on the folders.

**Comment by mawtex** [comment]
@Safety1st perhaps the solution @JustAndrei found works for you?

**Comment by Safety1st** [comment]
Is it right he did set rights to newly created folder?

**Comment by JustAndrei** [comment]
@Safety1st, I noticed you were trying to deploy the CMS on the hoster.by hosting, as well as I have been doing, so we've got the same problem. By default, hoster.by gives only read and execute rights to the app pool user group (IWPG_yourlogin). That may be relevant to other hosting providers as well. So, since the C1 CMS uses the file system for storing content, it needs the write permission. Also it needs it during self-configuration upon the first start, that's why it fails to start. So, specifically for hoster.by, the access rights can be configured in the file manager. I edited the httpdocs folder properties and provided the IWPG_yourlogin group with all the permissions, with propagation to child folders and files, then the site successfully started.
